packages/client/src/components/interface/Navigation.tsx

```diff
@@ -95,7 +101,11 @@ export const WORKQUEUE_TABS = {
 
 const GROUP_ID = {
   declarationGroup: 'declarationGroup',
-  menuGroup: 'menuGroup'
+  menuGroup: 'menuGroup',
+  leaderBoards: 'leaderBoards',
```

Comment:
Could we use `leaderboards` capitalisation through these changes?

Reply:
like instead of `leaderBoards` it should be `leaderboards`?

Reply:
ok